driver/ppm_fillers.c

```diff
@@ -991,8 +987,8 @@ int f_proc_startupdate(struct event_filler_arguments *args)
 			args_len = mm->arg_end - mm->arg_start;
 
 			if (args_len) {
-				if (args_len > PAGE_SIZE)
-					args_len = PAGE_SIZE;
+				if (args_len > STR_STORAGE_SIZE)
```

Comment:
`STR_STORAGE_SIZE` is `PAGE_SIZE` but if one day they should diverge is better to use `STR_STORAGE_SIZE` everywhere